res/yaml/locale_EN.yml

```diff
@@ -23,36 +23,36 @@ Headers:
-  databaseImportFailure: "<yellow>The <gold><value><yellow> data could not be imported into your current database. See logs for details."
-  databaseImportStarted: "<yellow>The <gold><value><yellow> data import has started..."
-  databaseImportSuccess: "<yellow>The <gold><value><yellow> data have been successfully imported into your current database."
-  deadHorsesCleared: "<yellow>All your dead horses have been cleared from the database."
-  deadHorsesClearedOther: "<yellow>All <gold><player><yellow>'s dead horses have been cleared from the database."
+  claimsLimitReached: "<yellow>Your claims limit has been reached."
+  claimsLimitReachedOther: "<gold><player><yellow>'s claims limit has been reached."
+  databaseImportFailure: "<yellow>The <gold><value><yellow> data could not be imported into your current database. See logs for details."
+  databaseImportStarted: "<yellow>The <gold><value><yellow> data import has started..."
+  databaseImportSuccess: "<yellow>The <gold><value><yellow> data have been successfully imported into your current database."
+  deadHorsesCleared: "<yellow>All your dead horses have been cleared from the database."
+  deadHorsesClearedOther: "<yellow>All <gold><player><yellow>'s dead horses have been cleared from the database."
   favoriteAlreadySet: "<yellow>You have already defined <gold><horse><yellow> as favorite horse."
   favoriteAlreadySetOther: "<gold><player><yellow> has already defined <gold><horse><yellow> as favorite horse."
   favoriteEdited: "<gold><horse><yellow> is now your favorite horse."
   favoriteEditedOther: "<gold><horse><yellow> is now the favorite horse of <gold><player><yellow>."
   friendAdded: "<gold><player><yellow> has been added to your friends list."
-  friendAddedReverse: "<gold><player><yellow> has added you in his friends list."
+  friendAddedReverse: "<gold><player><yellow> has added you to their friends list."
   friendAlreadyAdded: "<gold><player><yellow> is already in your friends list."
-  friendList: "<yellow>Your friends list : <value>"
-  friendListOther: "<gold><player><yellow>'s friend list : <value>"
-  friendListReverse: "<yellow>Friends lists to which you belong : <value>"
-  friendListReverseOther: "<yellow>Friends lists to which <gold><player><yellow> belongs : <value>"
+  friendList: "<yellow>Your friends list: <value>"
+  friendListOther: "<gold><player><yellow>'s friend list: <value>"
+  friendListReverse: "<yellow>Friends lists to which you belong: <value>"
+  friendListReverseOther: "<yellow>Friends lists to which <gold><player><yellow> belongs: <value>"
   friendRemoved: "<gold><player><yellow> has been removed from your friends list."
   friendRemovedReverse: "<gold><player><yellow> has removed you from his friends list."
   horseAlreadyClaimed: "<yellow>You have already claimed this horse."
-  horseAlreadyOnSale: "<gold><horse><yellow> is already on sale."
+  horseAlreadyOnSale: "<gold><horse><yellow> is already for sale."

```

Comment:
Yes, I did notice this, however I saw no breaking change in the plugin's functionality changing just the chat message. But if you'd like to keep consistency I will gladly go through and change the references in the code too.